tests/tracing/export/test_mlflow_v3_exporter.py

```diff
@@ -39,37 +36,26 @@ def _flush_async_logging():
 
 
 @pytest.mark.parametrize("is_async", [True, False], ids=["async", "sync"])
-@pytest.mark.parametrize("experiment_id", [None, _EXPERIMENT_ID])
```

Comment:
The `experiment_id` is `None` case does not appear to be valid and produces a StartTraceV3 request with {"trace_location": {"type": "MLFLOW_EXPERIMENT", "mlflow_experiment": {"experiment_id": ""}}}}`

I confirmed that this doesn't work on Databricks either (the backend correctly rejects this payload because experiment ID is empty)

Reply:
Ah, `_get_experiment_id()` returns `None` if no active experiment has been set.